sidecar/config/config.go

```diff
@@ -146,6 +159,9 @@ type Config struct {
 	Supervise bool `yaml:"supervise"`
 
 	HealthChecks []HealthCheck `yaml:"healthchecks"`
+	HealthChecksCACertPath string `yaml:"healthchecks_ca_cert_path"`
```

Comment:
Specifically HealthChecksCACertPath allowed me to test with this instead of yaml:

``` yaml
helloworld-v1:
  image: amalgam8/a8-examples-helloworld-sidecar:v1
  volumes:
   - 'certs:/opt/my/certs'
  environment:
    - A8_REGISTRY_URL=http://registry:8080
    - A8_SERVICE=helloworld:v1
    - A8_REGISTER=true
    - A8_ENDPOINT_PORT=5000
    - A8_ENDPOINT_TYPE=https
    - A8_HEALTHCHECKS=https://localhost:5000/health
    - A8_HEALTHCHECKS_CA_CERT_PATH=/opt/my/certs/ca_trust.pem
    - A8_LOG_LEVEL=debug
  external_links:
    - registry
```